packages/web/src/common/store/account/sagas.js

```diff
@@ -97,21 +100,32 @@ function* onSignedIn({ payload: { account } }) {
   const sentry = yield getContext('sentry')
   const analytics = yield getContext('analytics')
   if (account && account.handle) {
-    let solanaWallet = ''
-    try {
-      solanaWallet = (yield call(
-        getRootSolanaAccount,
-        audiusBackendInstance
-      )).publicKey.toBase58()
-    } catch (e) {
-      console.error('Failed to fetch Solana root wallet during identify()', e)
-    }
-    // Set analytics user context
+    const libs = yield call(audiusBackendInstance.getAudiusLibs)
+    const web3User = yield call([libs.Account, libs.Account.getWeb3User])
+
     const traits = {
       isVerified: account.is_verified,
-      trackCount: account.track_count,
-      solanaWallet
+      trackCount: account.track_count
```

Comment:
Nit - for clarity/grok-ability might be better to initialize `traits` once we know `managerUserId` and `solanaWallet` rather than adding properties to `traits` later.

Reply:
That's fair, though they aren't both added in all cases. So the idea was to add the things we know will exist all the time, then either tack on the manager properties or the solana wallet.